CPC-COM[18]-Cache_cash.Altis/scripts/spawn_ia_pluto.sqf

```diff
@@ -241,6 +246,9 @@ if (CC_p_menace_aa == 1) then {
 		_veh = [_pos,_side,_type,_group,(random 360),["NONE",0,0]] call GDC_fnc_lucySpawnVehicle;
 		_group = _veh#0; _veh = _veh#1;
 		switch (CC_p_typeia) do {
+			case 2: { // VCOM
```

Comment:
Un moyen de faire en sorte que VCOM ignore un groupe ?

Reply:
Oui en utilisant
`this setVariable ["Vcm_Disable",true];`